server/config.ts

```diff
@@ -15,30 +15,36 @@ const nonSensitiveSharedConfig = {
 }
 
 const keysToPullFromEnv = [
-  'SophtronClientId',
-  'SophtronClientSecret',
-
-  'UcpAuthClientId',
-  'UcpAuthClientSecret',
-  'UcpAuthEncryptionKey',
-
-  'CryptoKey',
-  'CryptoIv',
-
-  'Demo',
-  'DefaultProvider',
-  'Port',
-
-  'HostUrl',
-  'WebhookHostUrl',
-  'LogLevel',
-
-  'Env',
-
-  'RedisServer',
-  'RedisCacheTimeSeconds',
-  'ResourcePrefix',
-  'ResourceVersion'
+  "CryptoKey",
+  "CryptoIv",
+
+  "AuthProvider", //supported: ucp, local, sophtron
```

Comment:
We're going to need an end to end test that makes sure our auth is working when using the local AuthProvider.

You'll need to add some secrets and environment variables to this repo. If you don't have access, then ping @emilysallstrom.

You'll also need to add an extra github action that runs the e2e tests with the local authprovider. For right now it can just rerun all the tests with local auth. In the future we might want to only run a subset of the e2e tests with the local provider.